DEVELOPING.md

```diff
@@ -0,0 +1,8 @@
+# Developing Service-Workbench-on-AWS
+
+Put your notes on developing the package here.
+
+### Using Peru:
+
+* https://builderhub.corp.amazon.com/docs/brazil/user-guide/getting-started-peru.html
```

Comment:
Probably shouldn't publish an internal link!

Reply:
Removed it, refresh to see if it goes away